pkg/deployment/service.go

```diff
@@ -64,87 +58,3 @@ func GetService(ctx context.Context, helper *helper.Helper, service string) (dat
 	err := client.Get(ctx, types.NamespacedName{Name: service, Namespace: namespace}, foundService)
 	return *foundService, err
 }
-
-// EnsureServices - ensure the OpenStackDataPlaneServices exist
-func EnsureServices(ctx context.Context, helper *helper.Helper, instance *dataplanev1.OpenStackDataPlaneNodeSet) error {
```

Comment:
If you drop this, then where are you creating the services? Are you expecting the user to do that, if so, where do they get the YAML files?